cmd/res/configuration.yaml

```diff
@@ -0,0 +1,47 @@
+# REFERENCE: https://docs.edgexfoundry.org/3.0/microservices/device/Ch-DeviceServices/
+Writable:
+  LogLevel: INFO
+  Reading:
+    ReadingUnits: true
+  Telemetry:
+    Metrics:
+      EventsSent: false
+      ReadingsSent: false
```

Comment:
Remove, these are in the new common config here: https://github.com/edgexfoundry/edgex-go/blob/main/cmd/core-common-config-bootstrapper/res/configuration.yaml#L106-L111

Reply:
Removed